WBooks/src/redux/auth/actions.js

```diff
@@ -41,5 +41,8 @@ export const actionCreators = {
     const client = await asyncStorageOperations.getClient();
     const userId = await asyncStorageOperations.getUserId();
     login.setHeader(accessToken, client, userId);
+  },
+  removeError: () => dispatch => {
+    dispatch(privateActionsCreators.loginFailure({}));
```

Comment:
You should dispatch failure actions here, you should create an action that cleans the error

Reply:
But i'm dispatching failure actions there. Or Do I need to create a ResetAction and dispatch it?